3rd_party/glad/3.0/include/glad/glad.h

```diff
@@ -952,8 +952,8 @@ typedef void (APIENTRYP PFNGLDEPTHRANGEPROC)(GLdouble n, GLdouble f);
 GLAPI PFNGLDEPTHRANGEPROC glad_glDepthRange;
 #define glDepthRange glad_glDepthRange
 typedef void (APIENTRYP PFNGLVIEWPORTPROC)(GLint x, GLint y, GLsizei width, GLsizei height);
-GLAPI PFNGLVIEWPORTPROC glad_glViewport;
-#define glViewport glad_glViewport
+GLAPI PFNGLVIEWPORTPROC glad_glviewport;
```

Comment:
Anything in the `3rd_party` directory is library code, and therefore shouldn't be changed (unless when patching or modifying a library). Please revert all the changes in `3rd_party`.

Aside from that, `glViewport` is part of the OpenGL standard - it's an externally defined standard and not up to us to change it.